airflow/providers/amazon/aws/transfers/glacier_to_gcs.py

```diff
@@ -0,0 +1,122 @@
+        if any. For this to work, the service account making the request must have
+        domain-wide delegation enabled.
+    :type delegate_to: str
+    :param google_impersonation_chain: Optional Google service account to impersonate using
+        short-term credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    :type impersonation_chain: Union[str, Sequence[str]]
+    """
+
+    template_fields = ("vault_name", "bucket_name", "object_name")
+
+    @apply_defaults
+    def __init__(
+        self,
+        *,
+        aws_conn_id="aws_default",
+        gcp_conn_id="google_cloud_default",
+        vault_name: str,
+        bucket_name: str,
+        object_name: str,
+        gzip: bool,
+        chunk_size=1024,
+        delegate_to=None,
+        google_impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.aws_conn_id = aws_conn_id
+        self.gcp_conn_id = gcp_conn_id
+        self.vault_name = vault_name
+        self.bucket_name = bucket_name
+        self.object_name = object_name
+        self.gzip = gzip
+        self.chunk_size = chunk_size
+        self.delegate_to = delegate_to
+        self.impersonation_chain = google_impersonation_chain
+
+    def execute(self, context):
+        glacier_hook = GlacierHook(aws_conn_id=self.aws_conn_id)
+        gcs_hook = GCSHook(
+            gcp_conn_id=self.gcp_conn_id,
+            delegate_to=self.delegate_to,
+            impersonation_chain=self.impersonation_chain,
+        )
+        job_id = glacier_hook.retrieve_inventory(vault_name=self.vault_name)
+
+        with tempfile.NamedTemporaryFile() as temp_file:
+            glacier_data = glacier_hook.retrieve_inventory_results(
+                vault_name=self.vault_name, job_id=job_id["jobId"]
+            )
+            # Read the file content in chunks using StreamingBody
+            # https://botocore.amazonaws.com/v1/documentation/api/latest/reference/response.html
+            stream = glacier_data["body"]
+            for chunk in stream.iter_chunk(chunk_size=self.chunk_size):
+                temp_file.write(chunk)
+            temp_file.flush()
+            gcs_hook.upload(
+                bucket_name=self.bucket_name,
+                object_name=self.object_name,
+                filename=temp_file.name,
+                gzip=self.gzip,
+            )
+        return f"{self.bucket_name}/{self.object_name}"
```

Comment:
Will this be in form `gs://bucket_name/object_name`? Or `bucket_name/object_name`?

Reply:
`bucket_name/object_name`, is it ok?

Reply:
As a user I would expect valid gcs uri

Reply:
Ok. I will fix it.

Reply:
Commited